diff.py

```diff
@@ -1540,8 +1536,8 @@ def do_diff(basedump: str, mydump: str, config: Config) -> List[OutputLine]:
     for (line1, line2) in diff_lines(lines1, lines2, config.algorithm):
         line_color1 = line_color2 = sym_color = Format.NONE
         line_prefix = " "
-        out1 = Text.empty() if not line1 else Text.raw(pad_mnemonic(line1.original))
-        out2 = Text.empty() if not line2 else Text.raw(pad_mnemonic(line2.original))
+        out1 = Text() if not line1 else Text(pad_mnemonic(line1.original))
+        out2 = Text() if not line2 else Text(pad_mnemonic(line2.original))
```

Comment:
beware 8ffce9b63939a7329db3670c768a46360ea32b72 if you're moving `pad_mnemonic` early. I don't remember the details of that unfortunately so I can't say if it applies

Reply:
Seems like should be fine now, because it only adds whitespace to the mnemonic term (before it also padded the immediate)?